docs_src/versions.json

```diff
@@ -0,0 +1,11 @@
+[
```

Comment:
Looks like this being added to the `docs_src` folder. It currently exists under the `docs` folder
Per DevOps WG meeting recording, you thought it was existing in  `docs_src` folder

@ernestojeda , FYI. How will this impact the building of Docs? Will the existing in `Docs` folder suffice?

Reply:
@lenny-intel, I played around with the place of versions.json as it is important to the placement of the versions dropdown in the new MkDocs 1.5 (using Mike plugin to help provide this on the top bar like done in the OpenAMT project).  I have asked @ernestojeda to help me with this portion of the changes and we are still playing with it a bit.

@lenny-intel  Other than this is a change, do you have any issue of moving versions.json?

Reply:
@jpwhitemn This file is not needed here. When building the site without this file with `mkdocs build` the newer version selector worked fine.

Reply:
per meeting with @ernestojeda today - removed